docs/products/kafka/howto/manage-acls.rst

```diff
@@ -0,0 +1,53 @@
+Managing users and access control lists
+=======================================
+
+Aiven for Apache Kafka supports the definition of users and access control lists (ACL) to allow or limit producer and consumer rights at the topic.
+More information about permissions mapping can be found in the :doc:`dedicated documentation <../concepts/acl>`. 
+
+You can manage users and ACL entries in the corresponding tabs of the service page in the `Aiven web console <https://console.aiven.io/>`_.
+
+Add a user
+----------
+
+.. Warning:: 
+
+    A user with the ``Admin`` permission can create topics with any name, as the ``CreateTopics`` permission is applied at the cluster level. 
```

Comment:
Something seems to be suspicious about this phrase - is it really about "the name"? Or can create any topics inside a cluster? Because I thought the topic name is not limited at all for any user, who has permissions to create them.

Reply:
We have same phrase in "docs/products/kafka/concepts/acl", same question applies there :)

Reply:
Maybe this is what is meant "A user with the ``Admin`` permission is allowed to create any topics, because the ``CreateTopics`` permission is applied at the cluster level.  "